ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/enable-password.j2

```diff
@@ -1,3 +1,4 @@
+{# eos - Enable password #}
 {% if enable_password is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` does a deep test, this `if enable_password is arista.avd.defined` should be removed